package.json

```diff
@@ -1,84 +1,107 @@
+    "typeahead",
     "select",
-    "es2015"
+    "es2015",
+    "javascript",
+    "javascript-library",
+    "grunt",
+    "webpack",
+    "jest"
   ],
-  "author": "Isaac Suttell <isaac@isaacsuttell.com>",
+  "author": "Chris Orescan <chris.orescan@sony.com>",
+  "contributor": "Sepand Assadi <sepand.assadi@sony.com>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/ship-components/ship-components-select/issues"
+    "url": "https://github.com/ship-components/ship-components-tag-input/issues"
   },
-  "homepage": "https://github.com/ship-components/ship-components-select#readme",
+  "homepage": "https://github.com/ship-components/ship-components-tag-input#readme",
   "dependencies": {
+    "babel-runtime": "^6.23.0",
     "classnames": "^2.2.5",
-    "fuse.js": "^2.6.2",
-    "react": ">=15.0.0",
-    "react-dom": ">=15.0.0",
-    "ship-components-icon": "^0.1.1",
-    "ship-components-outsideclick": "^0.1.1"
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "ship-components-icon": "^0.1.2",
+    "ship-components-outsideclick": "^0.2.0",
+    "ship-components-utility": "^1.2.1"
   },
   "devDependencies": {
-    "autoprefixer": "^6.1.0",
-    "babel": "^6.0.15",
-    "babel-core": "^6.1.4",
-    "babel-eslint": "^4.1.5",
-    "babel-loader": "^6.1.0",
-    "babel-plugin-transform-es2015-classes": "^6.9.0",
-    "babel-plugin-transform-runtime": "^6.9.0",
-    "babel-preset-es2015": "^6.1.4",
-    "babel-preset-react": "^6.1.4",
-    "babel-preset-stage-2": "^6.11.0",
-    "css-loader": "^0.22.0",
-    "eslint": "^1.9.0",
-    "eslint-loader": "^1.1.1",
-    "eslint-plugin-react": "^3.8.0",
-    "extract-text-webpack-plugin": "^0.9.1",
-    "grunt": "^0.4.5",
-    "grunt-contrib-clean": "^0.6.0",
-    "grunt-eslint": "^17.3.1",
-    "grunt-webpack": "^1.0.11",
-    "jest-cli": "^14.1.0",
-    "object-assign": "^4.0.1",
-    "postcss-calc": "^5.0.0",
-    "postcss-color-function": "^2.0.0",
-    "postcss-color-hex-alpha": "^2.0.0",
-    "postcss-loader": "^0.7.0",
-    "postcss-nested": "^1.0.0",
-    "postcss-simple-vars": "^1.1.0",
-    "style-loader": "^0.13.0",
-    "webpack": "^1.12.6",
-    "webpack-dev-server": "^1.12.1"
+    "autoprefixer": "^7.1.1",
+    "babel-core": "^6.25.0",
+    "babel-eslint": "^7.2.3",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-react": "^6.24.1",
+    "coveralls": "^2.13.1",
+    "css-loader": "^0.28.4",
+    "eslint": "^4.1.1",
+    "eslint-config-ship-components": "^1.4.2",
+    "eslint-loader": "^1.8.0",
+    "eslint-plugin-react": "^7.1.0",
+    "extract-text-webpack-plugin": "^2.1.2",
+    "file-loader": "^0.11.2",
+    "grunt": "^1.0.1",
+    "grunt-contrib-clean": "^1.1.0",
+    "grunt-coveralls": "^1.0.1",
+    "grunt-eslint": "^20.0.0",
+    "grunt-webpack": "^3.0.0",
+    "jest": "^20.0.4",
+    "jest-css-modules": "^1.1.0",
+    "lodash": "^4.17.4",
+    "object-assign": "^4.1.1",
```

Comment:
We don't, I removed both from the code but forgot to uninstall  / remove from package